libs/bilge/src/Bilge/IO.hs

```diff
@@ -133,7 +136,7 @@ instance MonadHttp Wai.Session where
           , Wai.httpVersion = Client.requestVersion req
           , Wai.requestHeaders = Client.requestHeaders req
           , Wai.isSecure = Client.secure req
-          , Wai.remoteHost = error "no remote host"
+          , Wai.remoteHost = SockAddrInet 8080 (tupleToHostAddress (127, 0, 0, 1))  -- (this is fabricated, but leaving this undefined doesn't work.)
```

Comment:
Is this actually needed then?

Reply:
I don't think so. All tests passed for me without this change.

Reply:
I just opted for leaving it in to allow for future corner cases, but I don't have a strong case and I don't insist.

Reply:
I think in this case where we call the `Application` directly via `Wai.Session`, we don't need a remote host. There is no actual networking going on.
If for some reason this code starts doing network requests, it might even be better to find out because of an error instead of silently assuming an address. :man_shrugging:

Reply:
Yes, this was exactly @ChrisPenner point [back in the days](https://github.com/wireapp/wire-server/pull/724#discussion_r278115085), and @fisx agreed ;)

Reply:
I'll revert this commit and then wait for a last CI run to succeed.